Dockerfile

```diff
@@ -39,6 +39,10 @@ RUN yum install -y \
     && yum clean all && \
     rm -rf /var/cache/yum/* /tmp/* /var/tmp/*
 
+RUN yum module install -y \
+    # 'module' Crucible build dependencies
+    llvm-toolset
+
```

Comment:
We don't need llvm-toolset. 
`yum install clang -y` should work.

Reply:
Updated. If this looks good to you, I'll merge

Reply:
Looks good to me.